app/docker/views/containers/console/exec.html

```diff
@@ -51,16 +51,22 @@
             </div>
             <div class="form-group">
               <div class="col-sm-12">
-                <button type="button" class="btn btn-primary" ng-disabled="state.connected" button-spinner="state.connected" ng-click="connect()">
-                  <span ng-hide="state.leaveNetworkInProgress">Connect</span>
-                  <span ng-show="state.leaveNetworkInProgress">Connecting...</span>
+                <button type="button" class="btn btn-primary" ng-disabled="state.connecting || state.connected || !container.State.Running" ng-click="connectExec()">
```

Comment:
Replace `ng-disabled` with `ng-disabled="!container.State.Running"` as `state.connecting` and `state.connected` don't exist anymore.

Reply:
Done